nautobot_ssot_infoblox/utils/utilities.py

```diff
@@ -0,0 +1,61 @@
+"""Utilities for DiffSync related stuff."""
+
+from nautobot.extras.models import Tag
+from nautobot_ssot_infoblox.constant import TAG_COLOR
+
+
+def create_tag_sync_from_infoblox():
```

Comment:
I don't see any references to this code, I believe this function was removed with a recent PR from you.

Reply:
I've tested this and it appears that was a fluke in my previous testing and I can't recreate it. I've reverted the change so that the tags are applied to models with the `create_tag_sync_from_infoblox` function.